### PR TITLE
B&F Compete Profile 5 (Rate)

### PR DESCRIPTION
**Pull request by nadiia-tsytsylina**
## Describe your changes

Made request (send hourly rate) in 5th Complete Profile component; 
Add 5th component to main profile page
add setToken after successfull login

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/AC-73)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [ ] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/login-form/LoginForm.tsx [hunk 1]
```diff
@@ -47,7 +50,8 @@ function LoginForm(): JSX.Element {
     try {
       await signIn(data)
         .unwrap()
-        .then(() => {
+        .then(({ token }) => {
```
Comment (ProggerOleg) [inline src/components/login-form/LoginForm.tsx:53]: Можно наверное тип добавить, { token }: {token: string}

**Review by ProggerOleg — APPROVED** [review]

File: src/components/login-form/LoginForm.tsx (hunk 1 above)
Comment (lyudad) [reply src/components/login-form/LoginForm.tsx:53]: я бы не заметила, но с коментом согластна

Comment (nadiia-tsytsylina) [reply src/components/login-form/LoginForm.tsx:53]: я как-то этот коммент пропустила. увидела аппрув и не заметила.. сейчас исправлю

Comment (nadiia-tsytsylina) [reply src/components/login-form/LoginForm.tsx]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**